erpnext/crm/onboarding_step/email_group/email_group.json

```diff
@@ -0,0 +1,21 @@
+{
+ "action": "Go to Page",
+ "action_label": "Click here to learn more",
+ "creation": "2021-07-21 00:27:28.112277",
+ "description": "# Email Group\n\nAn Email Group is a list of subscriber email IDs to which you send newsletters. You can pull emails into Email Groups from various documents like Lead, Opportunity, Contacts, Employees to list a few.",
```

Comment:
If we click on the button, this is what happens since **Go to Page** actually expects a link from the system itself.

![image](https://user-images.githubusercontent.com/24353136/126440256-7e0efa87-8788-49b5-ad00-46f839894afc.png)

1. We can instead make this slide for Email Group creation and add a line in the description:
    You can read more about Email Groups in our documentation **here**.
2. Or we can fix this in the framework to allow external links to be opened in a new tab.